app/views/forumthreads/index.html.erb

```diff
@@ -0,0 +1,84 @@
+<%= link_to "Forums", forums_path %> →
+<% if params.to_hash.slice("label", "title", "content", "author", "reply").size > 0 %>
+  <%= link_to "All Threads", forumthreads_path %> → Search Results
+<% else %>
+  <%= "All Threads" %>
+<% end %>
+<% params_list = params.to_hash.slice("id", "query", "label", "title", "content", "author", "reply") %>
+<h1>
+  <% if params[:id] %>
+      <% text = "forum '#{Forum.find(params[:id]).name}'" %>
+  <% else %>
+      <% text = "all threads" %>
+  <% end %>
+  <% if params_list.size > 0 %>
+    <%= title "Search results in #{text} (#{@threads.length})" %>
+  <% else %>
+    <% if params[:id] %>
+      <%= title "All threads in #{text}" %>
+    <% else %>
+      <%= title "All Threads" %>
+    <% end %>
+  <% end %>
```

Comment:
I think it would be nice to include the use of `query`, `label`, `title`, `content`, `author`, and `reply` either in the title or below. Otherwise users need to look at the URL to figure out what the current search is.